llvm/lib/Support/CommandLine.cpp

```diff
@@ -2980,7 +2980,7 @@ void HelpPrinterWrapper::operator=(bool Value) {
 #ifdef SYCLomatic_CUSTOMIZATION
 void HelpPrinterWrapper::operator=(CtHelpCategory Value) {
   CategorizedPrinter.setReqCtHelpCategory(Value);
-  cl::HideUnrelatedOptions(CategorizedPrinter.getReqCtHelpCategory(Value));
+  // cl::HideUnrelatedOptions(CategorizedPrinter.getReqCtHelpCategory(Value));
```

Comment:
If it is not needed, please remove this line.

Reply:
Done